frontend/src/pages/MessagingQueues/MessagingQueues.tsx

```diff
@@ -63,12 +57,123 @@ function MessagingQueues(): JSX.Element {
 					</Button>
 				</div>
 			</div>
-			<div className="messaging-queue-main-graph">
-				<MessagingQueuesConfigOptions />
-				<MessagingQueuesGraph />
-			</div>
-			<div className="messaging-queue-details">
-				<MessagingQueuesDetails />
+			<div className="messaging-overview">
+				<p className="overview-text">
+					Start sending data in as little as 20 minutes
+				</p>
+				<p className="overview-subtext">Connect and Monitor Your Data Streams</p>
+				<div className="overview-doc-area">
+					<div className="overview-info-card">
+						<div>
+							<p className="card-title">Configure Consumer</p>
+							<p className="card-info-text">
+								Connect your consumer and producer data sources to start monitoring.
+							</p>
+						</div>
+						<div className="button-grp">
+							<Button
+								type="default"
+								onClick={(): void =>
+									history.push('/get-started/application-monitoring')
+								}
+								// todo-sagar check for cloud condition
```

Comment:
will this be taken separately ?

Reply:
yes, have started a thread with Ankit on slack